app/components/views/TransactionsPage/Header.js

```diff
@@ -0,0 +1,79 @@
+import { spring, Motion } from "react-motion";
+import { Link } from "react-router";
+import { injectIntl, defineMessages } from "react-intl";
+import "style/Header.less";
+import headerConnector from "connectors/transactionsPage";
+
+const opts = { stiffness: 150, damping: 15 };
```

Comment:
I feel like it's a bit bouncy now, after testing a bit `damping: 20` feels good.

Reply:
sure.. I got the numbers 150 and 15 from design but I'll change them